easybuild/tools/filetools.py

```diff
@@ -303,6 +364,47 @@ def download_file(filename, url, path):
         return None
 
 
+def make_tarfile(output_filename, source_dir):
+    """Make a tarfile with the output_filename of the source_dir"""
+    _log.debug('making new tarfile at %s from %s', output_filename, source_dir)
+    with tarfile.open(output_filename, "w:gz") as tar:
+            tar.add(source_dir, arcname=os.path.basename(source_dir))
+    _log.debug('done making tarfile at %s', output_filename)
+
+
+def download_svn(revision, url, path, timeout):
+    """
+    Download a svn repository and create a tarball out of it in the given path
+    timeout is ignored here
+    """
+    # import here to avoid circular import
+    from easybuild.tools.repository.svnrepo import SvnRepository
```

Comment:
to quote @stdweird: this indicates something is wrong in the way we organise things...

we got rid of all these inline imports, let's try not to switch back to them